dot/peerset/peerset_test.go

```diff
@@ -113,73 +211,186 @@ func TestPeerSetDiscovered(t *testing.T) {
 
 func TestReAllocAfterBanned(t *testing.T) {
 	t.Parallel()
-
```

Comment:
nit I personally like the empty line after `t.Parallel()` to separate the 'run this test in parallel' with the test body. But up to you and please don't change, it doesn't matter :wink: